snuba/datasets/cdc/groupedmessage.py

```diff
@@ -0,0 +1,60 @@
+from snuba.clickhouse import ColumnSet, DateTime, Nullable, String, UInt
+from snuba.datasets import DataSet
+from snuba.datasets.cdc.groupedmessage_processor import GroupedMessageProcessor
+from snuba.datasets.schema import ReplacingMergeTreeSchema
+
+
+class GroupedMessageDataSet(DataSet):
+    """
+    This is a full clone of postgres groupedmessage
+    generated through change capture.
+    """
+
+    def __init__(self):
+        columns = ColumnSet([
+            # columns to maintain the dataset
+            # for now this is actually the 32 bit commit_id, but we are trying to
+            # get the 64 bit one from postgres, thus we will create the column as
+            # 64 bit.
+            ('commit_id', UInt(64)),
+            # PG columns
+            ('id', UInt(64)),
+            ('logger', String()),
+            ('level', UInt(32)),
+            ('message', String()),
+            ('view', Nullable(String())),
+            ('status', UInt(32)),
+            #('times_seen', UInt(64)),
```

Comment:
These comments are here for the discussion in the code review. Once we are in agreement on which columns to keep I will remove the comments.